app/frontend/src/components/MapSearch.tsx

```diff
@@ -113,18 +131,19 @@ export default function MapSearch({
     const searchOption = searchOptions.find((o) => value === o.name);
 
     if (!searchOption) {
-      setValue(value);
+      // setAddress(value, value);
       //create-option is when enter is pressed on user-entered string
       if (reason === "create-option") {
         loadSearchOptions(value);
       }
     } else {
       setMarker(searchOption.location);
```

Comment:
Can we check for `(0, 0)` here and in that case not set the marker or anything? That way clicking the "no results" text won't take us there.

Reply:
Not sure what you mean? The no results text should be disabled?

Reply:
At the moment, if you click on "no results", I guess it flies to 0,0

Reply:
I can't replicate this, the option is disabled and I can't click on it